packages/evm/src/interpreter.ts

```diff
@@ -510,6 +565,10 @@ export class Interpreter {
      * @property {Uint8Array} memory the memory of the EVM as a `Uint8Array`
      * @property {BigInt} memoryWordCount current size of memory in words
      * @property {Address} codeAddress the address of the code which is currently being ran (this differs from `address` in a `DELEGATECALL` and `CALLCODE` call)
+     * @property {number} section the current EOF code section referenced by the PC
+     * @property {Uint8Array} immediate the immediate argument of the opcode
+     * @property {Uint8Array} error the error data of the opcode (only present for REVERT)
+     * @property {number} functionDepth the depth of the function call (only present for EOF)
```

Comment:
Storage is not added here yet

Reply:
Thinking about it, is this documentation necessary? Doesn't a dev immediately get the docs from the emitted event type?

Reply:
I wondered about that.  I think we need to move all of this to the `InterpreterStep` interface (or just remove it)